docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+On the other hand, `:Sex` is a single symbol, indicating that a single column vector should be extracted.
+Note that in the first case a vector is required to be passed (not just any iterable),
+so e.g. `german[:, (:Age, :Sex)]` is not allowed, but `german[:, [:Age, :Sex]]` is valid.
+
+### Most elementary get and set operations
+Here, "get" means you retrieve the column, and "set" means you put the column in the data frame.
+
+Given the data frame `german` here are various ways to grab one of its columns as vector:
+```jldoctest dataframe
+julia> german
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67  male        2  own      NA               little          ⋯
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> german.Age # all get the vector stored in our DataFrame without copying it
+1000-element Vector{Int64}:
+ 67
+ 22
+ 49
+ 45
+ 53
+ 35
+ 53
+ 35
+ 61
+ 28
+  ⋮
+ 34
+ 23
+ 30
+ 50
+ 31
+ 40
+ 38
+ 23
+ 27
+
+julia> german."Sex" # the same using string indexing
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+
+julia> german[:, 3] # note that this creates a copy
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+```
+
+To get one row as a `DataFrame`, we can index as follows:
```

Comment:
this is a section on selecting a column. Why do you discuss getting  a row in it?

Reply:
as commended - the section is about column selection. Why do you give an example of row selection in a section on selecting columns?

Reply:
Oh, ok. Now I got it. Above I have discussed the column selection. Now, the same operation I wanted to show for Row selection as well. So I have given an example for Row selection.

Reply:
I understand but why do you do it in the section of the manual on column selection?